[2023-09-17T09:55:45.115+0800] {processor.py:157} INFO - Started process (PID=86716) to work on /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T09:55:45.115+0800] {processor.py:829} INFO - Processing file /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py for tasks to queue
[2023-09-17T09:55:45.115+0800] {logging_mixin.py:151} INFO - [2023-09-17T09:55:45.115+0800] {dagbag.py:539} INFO - Filling up the DagBag from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T09:55:45.118+0800] {logging_mixin.py:151} INFO - [2023-09-17T09:55:45.117+0800] {dagbag.py:347} ERROR - Failed to import: /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py", line 8, in <module>
    'retry_delay' : timedelta(minute=2)
                    ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
[2023-09-17T09:55:45.119+0800] {processor.py:841} WARNING - No viable dags retrieved from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T09:55:45.193+0800] {processor.py:179} INFO - Processing /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py took 0.079 seconds
[2023-09-17T13:08:09.950+0800] {processor.py:157} INFO - Started process (PID=87025) to work on /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T13:08:09.950+0800] {processor.py:829} INFO - Processing file /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py for tasks to queue
[2023-09-17T13:08:09.950+0800] {logging_mixin.py:151} INFO - [2023-09-17T13:08:09.950+0800] {dagbag.py:539} INFO - Filling up the DagBag from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T13:08:09.953+0800] {logging_mixin.py:151} INFO - [2023-09-17T13:08:09.952+0800] {dagbag.py:347} ERROR - Failed to import: /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py", line 8, in <module>
    'retry_delay' : timedelta(minute=2)
                    ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
[2023-09-17T13:08:09.954+0800] {processor.py:841} WARNING - No viable dags retrieved from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T13:08:10.024+0800] {processor.py:179} INFO - Processing /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py took 0.075 seconds
[2023-09-17T16:14:03.427+0800] {processor.py:157} INFO - Started process (PID=87333) to work on /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T16:14:03.428+0800] {processor.py:829} INFO - Processing file /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py for tasks to queue
[2023-09-17T16:14:03.428+0800] {logging_mixin.py:151} INFO - [2023-09-17T16:14:03.428+0800] {dagbag.py:539} INFO - Filling up the DagBag from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T16:14:03.431+0800] {logging_mixin.py:151} INFO - [2023-09-17T16:14:03.430+0800] {dagbag.py:347} ERROR - Failed to import: /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py", line 8, in <module>
    'retry_delay' : timedelta(minute=2)
                    ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
[2023-09-17T16:14:03.431+0800] {processor.py:841} WARNING - No viable dags retrieved from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T16:14:03.502+0800] {processor.py:179} INFO - Processing /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py took 0.075 seconds
[2023-09-17T19:29:56.226+0800] {processor.py:157} INFO - Started process (PID=87631) to work on /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T19:29:56.226+0800] {processor.py:829} INFO - Processing file /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py for tasks to queue
[2023-09-17T19:29:56.227+0800] {logging_mixin.py:151} INFO - [2023-09-17T19:29:56.226+0800] {dagbag.py:539} INFO - Filling up the DagBag from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T19:29:56.230+0800] {logging_mixin.py:151} INFO - [2023-09-17T19:29:56.228+0800] {dagbag.py:347} ERROR - Failed to import: /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py", line 8, in <module>
    'retry_delay' : timedelta(minute=2)
                    ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
[2023-09-17T19:29:56.230+0800] {processor.py:841} WARNING - No viable dags retrieved from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T19:29:56.300+0800] {processor.py:179} INFO - Processing /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py took 0.075 seconds
[2023-09-17T23:08:26.161+0800] {processor.py:157} INFO - Started process (PID=88212) to work on /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T23:08:26.162+0800] {processor.py:829} INFO - Processing file /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py for tasks to queue
[2023-09-17T23:08:26.163+0800] {logging_mixin.py:151} INFO - [2023-09-17T23:08:26.163+0800] {dagbag.py:539} INFO - Filling up the DagBag from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T23:08:26.167+0800] {logging_mixin.py:151} INFO - [2023-09-17T23:08:26.165+0800] {dagbag.py:347} ERROR - Failed to import: /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py", line 8, in <module>
    'retry_delay' : timedelta(minute=2)
                    ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
[2023-09-17T23:08:26.167+0800] {processor.py:841} WARNING - No viable dags retrieved from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-17T23:08:26.546+0800] {processor.py:179} INFO - Processing /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py took 0.385 seconds
[2023-09-18T02:36:52.039+0800] {processor.py:157} INFO - Started process (PID=88631) to work on /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-18T02:36:52.040+0800] {processor.py:829} INFO - Processing file /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py for tasks to queue
[2023-09-18T02:36:52.040+0800] {logging_mixin.py:151} INFO - [2023-09-18T02:36:52.040+0800] {dagbag.py:539} INFO - Filling up the DagBag from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-18T02:36:52.042+0800] {logging_mixin.py:151} INFO - [2023-09-18T02:36:52.042+0800] {dagbag.py:347} ERROR - Failed to import: /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py", line 8, in <module>
    'retry_delay' : timedelta(minute=2)
                    ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
[2023-09-18T02:36:52.043+0800] {processor.py:841} WARNING - No viable dags retrieved from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-18T02:36:52.113+0800] {processor.py:179} INFO - Processing /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py took 0.074 seconds
[2023-09-18T05:56:17.012+0800] {processor.py:157} INFO - Started process (PID=88949) to work on /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-18T05:56:17.012+0800] {processor.py:829} INFO - Processing file /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py for tasks to queue
[2023-09-18T05:56:17.012+0800] {logging_mixin.py:151} INFO - [2023-09-18T05:56:17.012+0800] {dagbag.py:539} INFO - Filling up the DagBag from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-18T05:56:17.015+0800] {logging_mixin.py:151} INFO - [2023-09-18T05:56:17.014+0800] {dagbag.py:347} ERROR - Failed to import: /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py", line 8, in <module>
    'retry_delay' : timedelta(minute=2)
                    ^^^^^^^^^^^^^^^^^^^
TypeError: 'minute' is an invalid keyword argument for __new__()
[2023-09-18T05:56:17.016+0800] {processor.py:841} WARNING - No viable dags retrieved from /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py
[2023-09-18T05:56:17.086+0800] {processor.py:179} INFO - Processing /Users/homersimpson/Documents/GitHub/Rachel_Analysis_Project/Rachel_Analysis_Project/airflow_vitaminc/dags/vitaminc_dag.py took 0.075 seconds
